autoload/minimap/vim.vim

```diff
@@ -374,6 +375,70 @@ function! s:update_highlight() abort
         let pos = s:buffer_to_map(curr, total, mmheight)
         call s:highlight_line(winid, pos)
     endif
+
+    if g:minimap_git_colors
+        " Get git info
+        let git_call = 'git diff -U0 -- ' . expand('%')
+        let git_diff = substitute(system(git_call), '\n\+&', '', '') | silent echo strtrans(git_diff)
+
+        let lines = split(git_diff, '\n')
+        let diff_list = []
+        for line in lines
+            if line[0] ==? "@"
+                let this_diff = {}
+
+                let blobs = split(line, ' ')
+                let del_info = blobs[1]
+                let add_info = blobs[2]
+
+                " Parse newfile info
+                let add_info = split(add_info, ',')
+                let add_start = str2nr(add_info[0])
+                let add_len = 0
+                if len(add_info) > 1
+                    let add_len = abs(str2nr(add_info[1]))
+                endif
+                " Parse oldfile info
+                let del_info = split(del_info, ',')
+                let del_len = 0
+                if len(del_info) > 1
+                    let del_len = abs(str2nr(del_info[1]))
+                endif
+
+                " Get diff type + end line
+                let this_diff["start"] = add_start
+                let this_diff["end"] = this_diff["start"] + add_len
+                if add_len != 0 && del_len != 0
+                    let this_diff["color"] = g:minimap_diff_color
+                elseif add_len != 0
+                    let this_diff["color"] = g:minimap_diffadd_color
+                elseif del_len != 0
+                    let this_diff["color"] = g:minimap_diffremove_color
+                    let this_diff["end"] = this_diff["start"] + 1
+                else
+                    let this_diff["color"] = g:minimap_diff_color
+                    let this_diff["end"] = this_diff["start"] + 1
+                endif
+
```

Comment:
A minor note: please consider changing these double-quoted strings to single-quoted strings.  It matches the workflow's lint, matches the rest of the code, and is a little safer.